internal/standby/standby_test.go

```diff
@@ -1,39 +1,76 @@
-package standby_test
+package standby
```

Comment:
why do we need to use whitebox testing? couldn't we keep the blackbox approach?

Reply:
As noted [above](https://github.com/EvergenEnergy/remote-standby/pull/9#discussion_r1604296589), I changed this so I could make the struct fields private. Feels like a choice between two bad options 🤷‍♀️  The approach I had before wasn't really blackbox as it was accessing fields/methods that were only made public so they could be tested.

Reply:
by exposing the internals of your test, you are strongly coupling your test to the implementation details of your code. We should aim at keeping them as decoupled as possible, where the test tests the overall behaviour, allowing us to change implementation details without needing to update tests.

Reply:
If the test has no access to implementation details, I can only test this behaviour indirectly via an integration test with an MQTT client running. I feel it's worth the tradeoff of having the test closely coupled with the package, in order to be able to test the outage detection logic in a fast and simple way.

Reply:
ok the test is back in its own package